### ISSUE TITLE
Remove usage of deprecated & internal API calls

### ISSUE THREAD

**Issue by jothepro** [cross-language-cpp/djinni-intellij-plugin#12]
The Compatibility verification of the newly published plugin warns about
> Compatible. 9 usages of deprecated API. 1 usage of internal API

As a maintainer I'd like to update the plugin to not use internal & deprecated API calls, to reduce the risk of incompatibility in the future.

**Comment by jothepro** [comment]
the situation is getting more serious, as the scan from this week shows:

> Build of the IDE: IU-211.6693.111
Verdict: Compatible. 3 usages of scheduled for removal API and 8 usages of deprecated API. 1 usage of internal API

**Comment by a4z** [comment]
do they also report what API calls that are? 
or just the number, because only the number is not very helpful ...

**Comment by jothepro** [comment]
<img width="1395" alt="Bildschirmfoto 2021-04-15 um 18 52 28" src="https://user-images.githubusercontent.com/21294002/114907796-bd2bc500-9e1b-11eb-95f4-b636532e1415.png">

I don't know if the link is public: https://plugins.jetbrains.com/plugin/15347-djinni-idl-file-support/versions/stable/109422

**Comment by a4z** [comment]
no, I can not see that ,the link gives me just the usual download page
